MoltenVK/MoltenVK/GPUObjects/MVKPipeline.mm

```diff
@@ -1549,6 +1549,7 @@ static MTLVertexFormat mvkAdjustFormatVectorToSize(MTLVertexFormat format, uint3
 		}
 	}
 
+	shaderConfig.options.mslOptions.ios_support_base_vertex_instance = true;
```

Comment:
> However, I'm curious if a certain platform/metal version actually doesn't support this?

Good catch. You are right to question this. And yes, now that we've established this works in general, we should be a bit more formal when setting this parameter. 

Can you change the code to the following, which will take care of handling platform variations, please? 

```suggestion
	shaderConfig.options.mslOptions.ios_support_base_vertex_instance = getDevice()->_pMetalFeatures->baseVertexInstanceDrawing;
```